framework/util/spirv_parsing_util.cpp

```diff
@@ -644,52 +644,5 @@ std::vector<SpirVParsingUtil::BufferReferenceInfo> SpirVParsingUtil::GetBufferRe
     return ret;
 }
 
-static VkDescriptorType SpvReflectToVkDescriptorType(SpvReflectDescriptorType type)
```

Comment:
The same function is also defined in `framework/decode/vulkan_replay_consumer_base.cpp` which can also be removed

Reply:
@panos-lunarg was your feedback integrated?  Can you resolve this thread?